src/ui/public/management/__tests__/section.js

```diff
@@ -0,0 +1,108 @@
+import expect from 'expect.js';
+
+import ManagementSection from 'ui/management/section';
+import IndexedArray from 'ui/indexed_array';
+
+describe('ManagementSection', () => {
+  describe('constructor', () => {
+    it('defaults display to id', () => {
+      const section = new ManagementSection('kibana');
+      expect(section.display).to.be('kibana');
+    });
+
+    it('exposes items', () => {
+      const section = new ManagementSection('kibana');
+      expect(section.items).to.beEmpty;
+    });
+
+    it('sets url based on path', () => {
+      const section = new ManagementSection('kibana', { path: 'foo' });
+      expect(section.url).to.be('#/management/foo');
+    });
+
+    it('assigns all options', () => {
+      const section = new ManagementSection('kibana', { description: 'test' });
+      expect(section.description).to.be('test');
+    });
+  });
+
+  describe('register', () => {
+    let section;
+
+    beforeEach(() => {
+      section = new ManagementSection('kibana');
+    });
+
+    it('returns a ManagementSection', () => {
+      expect(section.register('about')).to.be.a(ManagementSection);
+    });
+
+    it('provides a reference to the parent', () => {
+      expect(section.register('about').parent).to.eql(section);
```

Comment:
If you want to ensure `parent` is a reference to the same object, it would be stricter to assert: `expect(section.register('about').parent).to.be(section);`. `eql` only recursively asserts `==`.